stable/yugaware/templates/configs.yaml

```diff
@@ -411,11 +411,11 @@ data:
 
       - job_name: 'kube-state-metrics'
         static_configs:
-        - targets: ['kube-state-metrics.kube-system.svc.{{.Values.domainName}}:8080']
+        - targets: ['kube-state-metrics.kube-state.svc.{{.Values.domainName}}:8080']
```

Comment:
our docs are asking users to install it in kube-system - https://docs.yugabyte.com/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes/#install-kube-state-metrics 

Let's keep it consistent with that unless there's a reason to use a different namespace.

Reply:
Ahh. I did it for testing. I installed multiple versions of Kube-state-metrics for testing. I forget to revert it. Will update. Thanks for capturing it.